.gitignore

```diff
@@ -1,5 +1,5 @@
 .idea
```

Comment:
this file should look something like

```
.idea
node_modules/
dist/app/
npm-debug.log
.config.json
```

the `/` at the end could tell you that you meant a folder
also you can just ignore every log file like `*.log`